cmd/gogio/help.go

```diff
@@ -59,4 +59,8 @@ The -work flag prints the path to the working directory and suppress
 its deletion.
 
 The -x flag will print all the external commands executed by the gogio tool.
+
+The -signkey flag specifies the path of the keystore, used for signature for Android. 
+
+The -signpass flag specifies the password of the signkey, ignored if no key is provided. 
```

Comment:
Nit:

The -signpass flag specifies the password of the keystore, ignored if -signkey is not provided.